packages/arb-token-bridge-ui/src/components/TransactionHistory/CustomMessageWarning.tsx

```diff
@@ -0,0 +1,17 @@
+export const CustomMessageWarning = ({
+  show = true,
```

Comment:
Can we at the parent level use `condition && <CustomMessageWarning>...` instead?

Reply:
updated